configs/body3d/simple_baseline/h36m/linear_h36m_block2x1024_maxnorm.py

```diff
@@ -77,34 +77,34 @@
     dict(
         type='JointRelativization',
         item='target',
-        visible_item='target_visible',
+        # visible_item='target_visible',
```

Comment:
Reason of this change

Reply:
This was to prevent dimension mismatch between the pose and the visibility after global position restoration in temporal_regression_head.This is fixed in the next commit.